README.md

```diff
@@ -1,299 +1,294 @@
-### Configuration properties
-
-You can pass additional parameters to configure an animation in the animation configuration object.
-All animations should accept the following properties:
-
- * `name`: The name of an animation, ie. an element implementing `Polymer.NeonAnimationBehavior`.
- * `node`: The target node to apply the animation to. Defaults to `this`.
- * `timing`: Timing properties to use in this animation. They match the [Web Animations Animation Effect Timing interface](https://w3c.github.io/web-animations/#the-animationeffecttiming-interface). The
- properties include the following:
-     * `duration`: The duration of the animation in milliseconds.
-     * `delay`: The delay before the start of the animation in milliseconds.
-     * `easing`: A timing function for the animation. Matches the CSS timing function values.
-
-Animations may define additional configuration properties and they are listed in their documentation.
-
-<a name="configuration-multiple"></a>
-### Using multiple animations
-
-Set the animation configuration to an array to combine animations, like this:
+### `NeonAnimationRunnerBehavior`
 
+#### In a Polymer 3 element
 ```js
-animationConfig: {
-  value: function() {
-    return {
-      // fade-in-animation is run with a 50ms delay from slide-down-animation
-      'entry': [{
-        name: 'slide-down-animation',
-        node: this
-      }, {
-        name: 'fade-in-animation',
-        node: this,
-        timing: {delay: 50}
-      }]
-    }
+import {PolymerElement, html} from '@polymer/polymer';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {NeonAnimationRunnerBehavior} from '@polymer/neon-animation/neon-animation-runner-behavior.js';
+import '@polymer/neon-animation/animations/scale-down-animation.js';
+
+class SampleElement extends mixinBehaviors([NeonAnimationRunnerBehavior], PolymerElement) {
+  static get template() {
+    return html`
+      <div>this entire element will be animated after one second</div>
+    `;
   }
-}
-```
 
-<a name="configuration-encapsulation"></a>
-### Running animations encapsulated in children nodes
+  connectedCallback() {
+    super.connectedCallback();
 
-You can include animations in the configuration that are encapsulated in a child element that implement `Polymer.NeonAnimatableBehavior` with the `animatable` property.
+    // must be set here because properties is static and cannot reference "this"
+    this.animationConfig = {
+      // provided by neon-animation/animations/scale-down-animation.js
+      name: 'scale-down-animation',
+      node: this,
+    };
 
-```js
-animationConfig: {
-  value: function() {
-    return {
-      // run fade-in-animation on this, and the entry animation on this.$.myAnimatable
-      'entry': [
-        {name: 'fade-in-animation', node: this},
-        {animatable: this.$.myAnimatable, type: 'entry'}
-      ]
-    }
+    setTimeout(() => this.playAnimation, 1000);
```

Comment:
`this.playAnimation` -> `this.playAnimation()`, there's a handful of these in the README.